scripts/src/modules/analytics/add_unique_ids.js

```diff
@@ -0,0 +1,23 @@
+const add_unique_ids = () => {
+
+    const items = [
+        {selector: '.blog-embed', id_prefix: 'analytics-blog-embed'},
+        {selector: '.related-resources ul', id_prefix: 'analytics-related-resources'}
+    ];
```

Comment:
Would we also want to include the `record-embed-no-image` class? It's currently a component in `featured_record.html`. I've checked the Insights pages and I've only seen it used once but it can be used more than once and it currently has `analytics-record-embed-no-image` set as the `id` in the HTML.